vault/changelog.release.2022-07-19.md

```diff
@@ -0,0 +1,85 @@
+---
+id: r50ffhwvk6c1ykmbj0q187q
+title: 0.104
+desc: ''
+updated: 1658248392170
+created: 1658222480416
+---
+
+Dendron 0.104 has sprouted  🌱
+
+**Jump Notes in Chronological Order**: Navigate seamlessly between sibling notes in the hierarchy of the note open with the new `Dendron: Go Next Sibling` command. 
```

Comment:
**Chronological Sibling Navigation (for daily journal notes)**:  You can now navigate across month and year boundaries across your daily journals